lib/runner/replay-controller.js

```diff
@@ -46,17 +46,12 @@ _.assign(ReplayController.prototype, /** @lends ReplayController.prototype */{
         !item.parent() && item.setParent(context.originalItem);
 
         // construct payload for request
-        var payload = _.defaults({
+        var payload = _.merge({}, desiredPayload, context, {
```

Comment:
@shamasis is it safe to do this or reconstruct context object? I think this allows `ReplayController` to have less knowledge of what `context` is made of.